Model/Service/Facet/BuildWebFacetService.php

```diff
@@ -193,31 +193,27 @@ private function mapIncludeFilter(StoreInterface $store, IncludeFilters &$includ
                 $value = (bool)$item->getData('value');
                 break;
             default:
-                $this->logger->debugWithSource(
+                $this->debugWithSource(
                     sprintf(
                         'Cannot build include filter for "%s" frontend input type',
                         $frontendInput
-                    ),
-                    [],
-                    $this
+                    )
                 );
                 return;
         }
         try {
             $attributeCode = $attributeModel->getAttributeCode();
             if (!is_string($attributeCode)) {
-                $this->logger->debugWithSource(
-                    sprintf(
-                        'Cannot build include filter for "%s" attribute ',
-                        $attributeModel->getName()
-                    ),
-                    [],
-                    $this
+                $this->debugWithSource(
+                    sprintf('Cannot build include filter for "%s" attribute ', $attributeModel->getName())
                 );
                 return;
             }
             $this->mapValueToFilter($includeFilters, $store, $attributeCode, $value);
-        } catch (NostoException $e) {
+        } catch (FacetValueException $e) {
+            $this->debugWithSource(
+                sprintf('Cannot map filters, error message: %s', $e->getMessage())
+            );
             $this->logger->exception($e);
```

Comment:
@olsi-qose what is this pattern btw? Why do we first log the message and then the exception?